package.json

```diff
@@ -31,19 +31,19 @@
     "angular-cookies": "^1.8.0",
     "angular-mocks": "^1.8.0",
     "angular-sanitize": "^1.8.0",
-    "chroma-js": "^0.6.3",
+    "chroma-js": "^2.1.0",
```

Comment:
Actually not sure re this update. doesn't have any benefits, but increased in size a lot. See https://github.com/matomo-org/matomo/pull/16034#issuecomment-640496350